console/src/components/Services/Data/TablePermissions/Permissions.js

```diff
@@ -1758,6 +1761,46 @@ class Permissions extends Component {
         );
       };
 
+      const getBackendOnlySection = () => {
```

Comment:
This file has almost 2000 LOC and this function behaves as a component. Could you move it to a separate component?

Reply:
Can we do this in a separate PR? This file anyway has to be split into mutliple files. It has way too much context to do it right now.